File: async/python/func.py
import time

def sleep(name: str, seconds: int):
    print('非同期処理の開始 name: %s seconds: %d' % (name, seconds))
    time.sleep(seconds)
    print('非同期処理の終了 name: %s seconds: %d' % (name, seconds))
    return name

File: async/nodejs/src/func.ts
export function sleep(name: string, seconds: number) {
	return new Promise((resolve, reject) => {
		console.log(`非同期処理の開始 name: ${name} seconds: ${seconds}`);
		setTimeout(() => {
			console.log(`非同期処理の終了 name: ${name} seconds: ${seconds}`);
			resolve(name);
		}, seconds * 1000);
	});
}
